deploy/crds/kieapp.crd.yaml

```diff
@@ -4018,6 +4058,46 @@ spec:
                                     to generated options in JAVA_OPTS. e.g. '-Dsome.property=foo'
                                   type: string
                               type: object
+                            kafka:
```

Comment:
update the field descriptions to match the added on the templates.

Reply:
done